components/collapsible-panel/test/collapsible-panel.visual-diff.js

```diff
@@ -0,0 +1,54 @@
+/*global forceFocusVisible */
+import puppeteer from 'puppeteer';
+import VisualDiff from '@brightspace-ui/visual-diff';
+
+describe('d2l-collapsible-panel', () => {
+
+	const visualDiff = new VisualDiff('collapsible-panel', import.meta.url);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser);
+		await page.goto(`${visualDiff.getBaseUrl()}/components/collapsible-panel/test/collapsible-panel.visual-diff.html`, { waitUntil: ['networkidle0', 'load'] });
+		await page.bringToFront();
+	});
+
+	after(async() => await browser.close());
+
+	[
```

Comment:
I think we should have RTL versions of some of these.

`d2l-switch` has an example on how to manage this when there are several components in play. (see [html page](https://github.com/BrightspaceUI/core/blob/main/components/switch/test/switch.visual-diff.html#L6), see [javascript](https://github.com/BrightspaceUI/core/blob/main/components/switch/test/switch.visual-diff.js#L78)). The alternative is to setup the fixture with `dir="rtl"`, but that is a limited solution that only targets the host, so the solution I linked is better.

We're not totally consistent in how we organize the tests, but we do try to avoid running unnecessary tests (like check RTL for something that we would never expect to be affected by RTL).

Reply:
Good call. Yeah a RTL test probably makes sense here. I'll take a look.